src/renderer/store/slices/tabSlice.ts

```
@@ -297,24 +296,24 @@ export const createTabSlice: StateCreator<AppState, [], [], TabSlice> = (set, ge
+            conversation: cachedTabData.conversation,
+            conversationLoading: false,
+            sessionDetailLoading: false,
+            sessionDetailError: null,
+            sessionClaudeMdStats: cachedTabData.sessionClaudeMdStats,
+            sessionContextStats: cachedTabData.sessionContextStats,
+            sessionPhaseInfo: cachedTabData.sessionPhaseInfo,
+            visibleAIGroupId: cachedTabData.visibleAIGroupId,
+            selectedAIGroup: cachedTabData.selectedAIGroup,
+          });
+        } else {
+          void get().fetchSessionDetail(foundWorktree, sessionId, tabId);
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)